tools/vsimporter/src/vsimporter.cpp

```diff
@@ -282,15 +290,11 @@ int main(int argc, char* argv[])
   // Initialize global settings
   String binaryDir = sb_dirname(getBinaryLocation());
   sbValidateWithTelemetry(!binaryDir.empty(), "Failed to resolve binary directory.");
+  if (!templatesPath.empty()) {
+    settingsManager.setGlobalVar("VSIMPORTER_TEMPLATES_DIR", joinPaths(getcwd(), templatesPath));
```

Comment:
vsimporter.exe and templates will eventually live in a Tools package in some random not super easy to get to path. 

Full path to the templates is best if thats knowable ahead of time. Was thinking that at least we can know the relative directory structure between vsimporter.exe and the templates and can get the .exe path at runtime

---
In reply to: [100940407](https://github.com/Microsoft/WinObjC/pull/1958#discussion_r100940407) [](ancestors = 100940407,100936693)